node.gypi

```diff
@@ -256,20 +239,13 @@
         }],
       ],
     }],
-    [ 'node_engine=="v8"', {
+    ['node_engine=="chakracore"', {
```

Comment:
I've been building V8 and ChakraCore using both Linux and Windows.  They both build and the tests pass with some small amount of flakiness.